main.go

```diff
@@ -301,6 +302,7 @@ func init() {
 	// Setting RAM causes issues right now
 	sshCmd.Flags().IntVar(&config.Memory, "mem", 0, "Amount of RAM to allocate in GB, e.g., 16")
 	sshCmd.Flags().IntVar(&config.HDD, "hdd", 0, "Amount of hard-disk space to allocate in GB")
+	sshCmd.Flags().StringSliceVarP(&config.Volumes, "volume", "v", []string{}, "Mount a volumes to newly created execs. e.g., -v <volume-name>:/data")
```

Comment:
Typo: `Mount a volume` instead of `volumes`. This is my bad. Will also need changing in the `new` command